scripts/policy/protocols/ssl/decryption.zeek

```diff
@@ -0,0 +1,106 @@
+#! Decrypt SSL/TLS payloads
+
+@load base/frameworks/input
+@load base/frameworks/notice
+@load base/protocols/conn
+@load base/protocols/ssl
+
+module SSL;
+
+# Local
+const input_stream_name = "input-tls-keylog-file";
+
+type Idx: record {
+	client_random: string;
+};
+
+type Val: record {
+	secret: string;
+};
+
+global randoms: table[string] of string = {};
+
+export {
```

Comment:
Small style not - we typically have the export section before any other type declarations. Though I can just fix that while merging

Reply:
I tried to do that across the codebase, but I seem to have missed some places. I will make another pass soon and resolve this.